vmngclient/api/basic_api.py

```diff
@@ -175,19 +150,29 @@ def wait_for_state():
 
         return True if wait_for_state() else False
 
-    def get(self, field: DeviceField, value: str) -> Device:
-        supported_fields = [
-            DeviceField.HOSTNAME,
-            DeviceField.ID,
-        ]
+    def get_devices(self, rediscover: bool = False) -> DataSequence[Device]:
+        """Data sequence of all devices.
+
```

Comment:
Please include optional argument usage in docstring. This is very much needed and instructive for this case!